app.js

```diff
@@ -0,0 +1,39 @@
+//#region SERVER SETUP
+
+const { exec } = require('child_process');
+const express = require('express'),
+    app = express(),
+    port = 3000;
+
+// Express Middleware for serving static files.
+app.use(express.static(__dirname + '/public'));
+
+//Kicks off the app on a specified port
+app.listen(port);
+console.log(`Your server available at http://localhost:${port}`);
+
+//#endregion
+
+//#region ROUTES
+
+//The landing page of the site. 
+app.get('/', function (req, res) {
+    res.redirect('index.html');
```

Comment:
Going to the default route, in this case `localhost:3000` will return the index.html file. By adding more routes and `get` request handles you can add more pages to the site.